examples/memory_basic_usage.py

```diff
@@ -0,0 +1,133 @@
+"""
+========================
+How to use joblib.Memory
+========================
+
+This example illustrates the usage of :class:`joblib.Memory`. We illustrate
```

Comment:
Like some previous PR on documentation, it would be nice to avoid using `We` or `You` and try to be more neutral.

Reply:
We taught this rule during my PhD but I am 50/50 after reading Writing Science" by J. Schimel. I would not change `we` for a passive voice. It makes the sentence more "stuffy" and remove some dynamic. However, it is possible that there is a better sentence in the active voice by changing the agent of the sentence and avoid `we`.

Actually, I will be interested by the opinion of @GaelVaroquaux who have some other references than mine on how to write (which I did not read yet :S)